mocks/github.com/vektra/mockery/v2/pkg/fixtures/buildtag/comment/IfaceWithBuildTagInComment.go

```diff
@@ -0,0 +1,87 @@
+//go:build custom && (!windows || !linux || !darwin || !freebsd)
+
+// Code generated by mockery. DO NOT EDIT.
```

Comment:
We need to double check if this `DO NOT EDIT` comment being placed below the first line still meets the specification: https://github.com/golang/go/issues/41196

Reply:
Thanks for pointing it out, I will fix it.

Reply:
Fixed

Reply:
By the way [boilerplate text](https://github.com/vektra/mockery/blob/138d507e2b506420f49969570b816f31287b8bed/pkg/generator.go#L112C4-L112C23) is generated before `// Code generated ...`. Is it correct?